src/Security/Authentication/Certificate/src/README.md

```diff
@@ -0,0 +1,239 @@
+# Microsoft.AspNetCore.Authentication.Certificate
```

Comment:
Move to a doc PR/bug?

Reply:
Yeah docs with this will come later, readme seems ok to leave in the meantime